### ISSUE TITLE
A11y: Collapsed Service Hotline on mobile cannot be reached with a keyboard

### ISSUE THREAD

**Issue by shopwareBot** [shopware/platform#6995]
The only way to expand this is with a mouse, which is a failure of SC: 2.1.1
[https://www.w3.org/WAI/WCAG22/Techniques/failures/F54.html|https://www.w3.org/WAI/WCAG22/Techniques/failures/F54.html|smart-link] 

This also means it cannot be accessed with a screen reader, or any other sort of assistive technology.

!Screenshot 2025-02-21 at 16.51.57.png|width=371,height=664,alt="Screenshot 2025-02-21 at 16.51.57.png"!

**Comment by shopwareBot** [comment]
[public] 

 > Priority changed to: **High**

 Thank you for your report. Based on the information in this report, we have decided to investigate this issue in more detail and are planning to provide a fix for this issue. Although we are always eager to solve such problems as soon as we can, we can't give you a more specific time frame right now. We appreciate your patience and thank you for your understanding.



[created from NEXT-40777, comment 540977]

**Comment by shopwareBot** [comment]
[public] Shopware automated response: This issue is linked to the internal issue: **NEXT-40777**



[created from NEXT-40777, comment 540982]

**Comment by shopwareBot** [comment]
[public] 

 > FixVersion Change: **6.6.10.3**

 This issue will be / was solved in Shopware version ***6.6.10.3***.



[created from NEXT-40777, comment 543183]
